manifests/q/Questrade/IQEdge/6.6.44.127/Questrade.IQEdge.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Automatically updated by the winget bot at 2022/Dec/10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Questrade.IQEdge
+PackageVersion: 6.6.44.127
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Based on the validation error, I believe the architecture might have changed between versions. 
![image](https://user-images.githubusercontent.com/114952565/207100489-54dd8e55-eb65-4bae-8e7c-bc94b93f2aab.png)